codegen_glibmm/templates/stub.cpp.templ

```diff
@@ -177,9 +178,10 @@ bool {{ class_name_with_namespace }}::on_interface_set_property(
 
 {% for signal in interface.signals if signal is supported_by_sigc %}
 void {{ class_name_with_namespace }}::{{ signal.name }}_emitter(
+    const std::vector<Glib::ustring> &destination_bus_names,
 {%- set comma = joiner() -%}
 {%- for arg in signal.args -%}
-{{ comma() }}{{ arg.cpptype_out }} {{ arg.name }}
+{{ comma() }}{{ arg.cpptype_out }} &{{ arg.name }}
```

Comment:
The `&` should not be here. I suspect that the `arg.cpptype_out` is wrong, and that it should rather be `arg.cpptype_in`. But this is not really related with your changes, so, if you fix this, please do it in a separate commit.

Reply:
Sure I will fix it to separate commit

Reply:
I also patched other commit to this PR